src/firebolt/async_db/cursor.py

```diff
@@ -271,6 +270,10 @@ async def executemany(
         Prepare and execute a database query against all parameter
         sequences provided. Return last query row count.
         """
+
+        if len(parameters_seq) > 1:
+            raise NotSupportedError("multi-statement queries are not supported")
```

Comment:
Should this be "Parametrised multi-statement queries are not supported"?

Reply:
Yeah, we can use that